DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/ContractDetails/_ContractDetailsList.cshtml

```diff
@@ -0,0 +1,92 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.ContractDetails
+@model ContractDetailsViewModel
+
+<ul class="nhsuk-grid-row nhsuk-card-group">
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+    <div class="nhsuk-card">
+      <div class="nhsuk-card__content">
+        <p class="nhsuk-heading-xl nhsuk-u-font-size-64 nhsuk-u-margin-bottom-1 coloured-card-number--blue">
+          @Model.Administrators
+        </p>
+        <span class="nhsuk-u-font-weight-normal nhsuk-u-font-size-19">Administrators</span>
+      </div>
+    </div>
+  </li>
+
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+    <div class="nhsuk-card">
+      <div class="nhsuk-card__content">
+        <p class="nhsuk-heading-xl nhsuk-u-font-size-64 nhsuk-u-margin-bottom-1 @("coloured-card-number--" + Model.CmsAdministratorsColour)">

```

Comment:
Ah - I think Steve went to an accessibility session, where it was said that p tag should only be used for true paragraphs of text. Maybe we can use span instead?

Reply:
This is the html from the NHS component, do we want to change it?

Reply:
Let's keep it for now and if it causes any issues change it across the site 👍